src/Test.UnitTests.Sarif.Converters/TestData/FortifyFprConverter/ExpectedOutputs/OneResultWithTwoTraces.sarif

```diff
@@ -354,10 +354,17 @@
               "fullDescription": {
                 "text": "The program can potentially fail to release a system resource.\r\n\r\nResource leaks have at least two common causes:\r\n\r\n- Error conditions and other exceptional circumstances.\r\n\r\n- Confusion over which part of the program is responsible for releasing the resource.\r\n\r\nIn this case, there are program paths on which the resource allocated in <Replace key=\"FirstTraceLocation.file\"/> at line <Replace key=\"FirstTraceLocation.line\"/> is not released.\r\n\r\nMost unreleased resource issues result in general software reliability problems, but if an attacker can intentionally trigger a resource leak, the attacker may be able to launch a denial of service attack by depleting the resource pool.\r\n\r\n**Example:** Under normal conditions the following code executes a database query, processes the results returned by the database, and closes the allocated `SqlConnection` object. But if an exception occurs while executing the SQL or processing the results, the `SqlConnection` object will not be closed. If this happens often enough, the database will run out of available cursors and not be able to execute any more SQL queries.\r\n\r\n`\n        ...\n        SqlConnection conn = new SqlConnection(connString);\n        SqlCommand cmd = new SqlCommand(queryString);\n        cmd.Connection = conn;\n        conn.Open();\n        SqlDataReader rdr = cmd.ExecuteReader();\n        HarvestResults(rdr);\n        conn.Connection.Close();\n        ...\n`"
               },
+              "defaultConfiguration": {
```

Comment:
>defaultConfiguration [](start = 15, length = 20)

Since we are now assigning the computed `level` both to `result.level` and to `rule.defaultConfiguration.level`, I'm surprised I don't see `level` here.

Reply:
If level is computed as "Warning" then it doesnot shows up in sarif.

Reply:
... ah. Ok. Are there any example test files where level is _not_ warning, so we do see it here?

---
In reply to: [314903143](https://github.com/microsoft/sarif-sdk/pull/1650#discussion_r314903143) [](ancestors = 314903143)

Reply:
example file: ScanWithFailureLevelMatrices.sarif